cosmosdb/create-and-configure-cassandra-database/create-and-configure-cassandra-database.ps1

```diff
@@ -1,44 +0,0 @@
-# Set the Azure resource group name and location
```

Comment:
Cannot find this article referenced in published docs. I see the article itself but do not see how to navigate to it.

Reply:
The docs are published but looks like they aren't linked to from the navigator page. It's here:

https://docs.microsoft.com/en-us/azure/cosmos-db//scripts/powershell/create-and-configure-cassandra-database?toc=%2fpowershell%2fmodule%2ftoc.json

Reply:
Yes the docs are not linked but users can still find them with browser search because they are published. If we don't want them we should delete the .md files from the repo.